### PR TITLE
ci: update pullapprove config to add comp labels

### PR DESCRIPTION
**Pull request by jessicajaniuk**
This adds comp labels to each group defined in the yml file.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

No labels are applied automatically when pullapprove runs

Issue Number: N/A

## What is the new behavior?
PullApprove will auto apply component labels for each PR that's created.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jessicajaniuk** [comment]
/pullapprove-test

```yml
version: 3
groups:
  fw-router:
    conditions:
      - >
        contains_any_globs(files, [
          'packages/router/**',
          'packages/examples/router/**',
          'aio/content/guide/router.md',
          'aio/content/guide/router-tutorial.md',
          'aio/content/examples/router-tutorial/**',
          'aio/content/examples/router/**',
          'aio/content/images/guide/router/**'
          ])
    labels: "comp: router"
    reviewers:
      users:
        - atscott
```

**Review by jessicajaniuk — COMMENTED** [review]
/pullapprove-test

```yml
version: 3
groups:
  fw-router:
    conditions:
      - >
        contains_any_globs(files, [
          'packages/router/**',
          'packages/examples/router/**',
          'aio/content/guide/router.md',
          'aio/content/guide/router-tutorial.md',
          'aio/content/examples/router-tutorial/**',
          'aio/content/examples/router/**',
          'aio/content/images/guide/router/**'
          ])
    labels: "comp: router"
    reviewers:
      users:
        - atscott
```

**Review by jessicajaniuk — COMMENTED** [review]
/pullapprove-test

version: 3
groups:
  fw-router:
    conditions:
      - >
        contains_any_globs(files, [
          'packages/router/**',
          'packages/examples/router/**',
          'aio/content/guide/router.md',
          'aio/content/guide/router-tutorial.md',
          'aio/content/examples/router-tutorial/**',
          'aio/content/examples/router/**',
          'aio/content/images/guide/router/**'
          ])
    labels:
      "comp: router": null
    reviewers:
      users:
        - atscott

**Review by jessicajaniuk — COMMENTED** [review]
/pullapprove-test

```yml
version: 3
groups:
  fw-router:
    conditions:
      - >
        contains_any_globs(files, [
          'packages/router/**',
          'packages/examples/router/**',
          'aio/content/guide/router.md',
          'aio/content/guide/router-tutorial.md',
          'aio/content/examples/router-tutorial/**',
          'aio/content/examples/router/**',
          'aio/content/images/guide/router/**'
          ])
    labels:
      "comp: router": null
    reviewers:
      users:
        - atscott
```

**Review by jessicajaniuk — COMMENTED** [review]
/pullapprove-test

```yml
version: 3
groups:
  fw-router:
    conditions:
      - >
        contains_any_globs(files, [
          'packages/router/**',
          'packages/examples/router/**',
          'aio/content/guide/router.md',
          'aio/content/guide/router-tutorial.md',
          'aio/content/examples/router-tutorial/**',
          'aio/content/examples/router/**',
          'aio/content/images/guide/router/**'
          ])
    labels:
      pending: "comp: router"
    reviewers:
      users:
        - atscott
```

**Review by jessicajaniuk — COMMENTED** [review]
/pullapprove-test

```yml
version: 3
groups:
  fw-router:
    conditions:
      - >
        contains_any_globs(files, [
          'packages/router/**',
          'packages/examples/router/**',
          'aio/content/guide/router.md',
          'aio/content/guide/router-tutorial.md',
          'aio/content/examples/router-tutorial/**',
          'aio/content/examples/router/**',
          'aio/content/images/guide/router/**'
          ])
    labels:
      pending: "comp: router"
      approved: "comp: router"
      rejected: "comp: router"
    reviewers:
      users:
        - atscott
```

**Review by jessicajaniuk — COMMENTED** [review]
/pullapprove-test

```yml
version: 3
groups:
  code-ownership:
      conditions:
          contains_any_globs(files, [
            '.pullapprove.yml'
            ])
      labels:
        pending: "comp: build & ci"
        approved: "comp: build & ci"
        rejected: "comp: build & ci"
      reviewers:
        users:
          - jessicajaniuk
```

**Review by jessicajaniuk — COMMENTED** [review]
/pullapprove-test

```yml
version: 3
groups:
  code-ownership:
    conditions:
      - >
        contains_any_globs(files, [
          '.pullapprove.yml'
          ])
    labels:
      pending: "comp: build & ci"
      approved: "comp: build & ci"
      rejected: "comp: build & ci"
    reviewers:
      users:
        - jessicajaniuk
```

File: .pullapprove.yml [hunk 1]
```diff
@@ -37,7 +37,6 @@
 # See reviewer list under `required-minimum-review` group.  Team member names and
 # usernames are managed there.
 
-
```
Comment (josephperrott) [inline .pullapprove.yml:40]: I think you have yaml formatter that changed the format of the file?

**Review by josephperrott — CHANGES_REQUESTED** [review]
Overall I think its good, other than the formatting, but we should have jelbourn review as well.

File: .pullapprove.yml (hunk 1 above)
Comment (jessicajaniuk) [reply .pullapprove.yml:40]: Crap. Yes it changed more than I thought. I'll address that.

Comment (jessicajaniuk) [reply .pullapprove.yml]: OK, formatting should be addressed now.

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by jelbourn — APPROVED** [review]
LGTM

File: .pullapprove.yml [hunk 2]
```diff
@@ -435,6 +453,34 @@ groups:
         # OOO as of 2020-09-28 - pkozlowski-opensource
 
 
+  # =========================================================
+  #  Framework: Common
+  # =========================================================
```
Comment (petebacondarwin) [inline .pullapprove.yml:458]: How did we not have an `fw-common` group before??

Comment (gkalpak) [reply .pullapprove.yml:458]: It was merged with `fw-core` (i.e. owned by the same people), but now we need different labels :grin:

**Pull request closed without merging**